notation.go

```diff
@@ -325,3 +390,17 @@ func generateAnnotations(signerInfo *signature.SignerInfo) (map[string]string, e
 		annotationX509ChainThumbprint: string(val),
 	}, nil
 }
+
+func (outcome *VerificationOutcome) GetUserMetadata() (map[string]string, error) {
```

Comment:
According to [effective go](https://go.dev/doc/effective_go#Getters), the `Get` prefix should be removed.
```suggestion
func (outcome *VerificationOutcome) UserMetadata() (map[string]string, error) {
```